src/release/breaking-changes/eliminating-nullok-parameters.md

```diff
@@ -54,7 +54,7 @@ return a non-nullable value:
 * [`FocusTraversalOrder.of`][]
 * [`FocusTraversalGroup.of`][]
 * [`Focus.of`][]
-* [`Shortcuts.of`][]
```

Comment:
Does `Shortcuts.of` not have a target page like everything else in this list? That seems... unexpected?

Reply:
The `Shortcuts` methods have been removed in a more recent version. Perhaps I should add a small note in parenthesis providing some extra context.

Reply:
SGTM. I'm assuming if it is removed in `master` that it was deprecated previously. Maybe Something along the lines of "Deprecated, to be removed in version XXX" or similar.